documentation/modules/cruise-control/proc-automating-rebalances.adoc

```diff
@@ -0,0 +1,188 @@
+When auto-rebalancing is enabled, the cluster is rebalanced without further intervention.
+
+* After adding brokers, topic partitions are redistributed across the new brokers.
+* Before removing brokers, partitions are moved off the brokers being removed.
+
+Auto-rebalancing helps maintain balanced load distribution across Kafka brokers during scaling operations, depending on how the rebalancing configuration is set up.
+
+Scaling operates in two modes: `add-brokers` and `remove-brokers`.
+Each mode can have its own auto-rebalancing configuration specified in the `Kafka` resource under `spec.cruiseControl.autoRebalance` properties. 
+Use the `template` property to specify a predefined `KafkaRebalance` resource, which serves as a rebalance configuration template.
+If a template is not specified in the `autorebalance` configuration, the default Cruise Control rebalancing configuration is used.
+You can apply the same template configuration for both scaling modes, use different configurations for each, or enable auto-rebalancing for only one mode.
+If `autorebalance` configuration is not set for a mode, auto-rebalancing will not occur for that mode.
+
+The template `KafkaRebalance` resource must include the `strimzi.io/rebalance-template: true` annotation. 
+The template does not represent an actual rebalance request but holds the rebalancing configuration.
+During scaling, the Cluster Operator creates a `KafkaRebalance` resource based on this template, named `<cluster_name>-auto-rebalancing-<mode>`, where <mode> is either `add-brokers` or `remove-brokers`. 
+The Cluster Operator applies a finalizer (`strimzi.io/auto-rebalancing`) to prevent the resource's deletion during the rebalancing process.
+
+Progress is reflected in the status of the `Kafka` resource.
+The `status.autoRebalance` property indicates the state of the rebalance. 
+A `modes` property lists the brokers being added or removed during the operation to help track progress across reconciliations.
+
+.Prerequisites
+
+* xref:deploying-cluster-operator-str[The Cluster Operator must be deployed.]
+* xref:proc-configuring-deploying-cruise-control-str[Cruise Control is deployed with Kafka.]
+* You have configured optimization goals and, optionally, capacity limits on broker resources.
+
+.Procedure
+
+. Create a rebalancing template for the auto-rebalancing operation (if required).
++
+Configure a `KafkaRebalance` resource with the `strimzi.io/rebalance-template: true` annotation.
+The rebalance configuration template does not require `mode` and `brokers` properties unlike when xref:#proc-generating-optimization-proposals-str[when generating an optimization proposal for rebalancing]. 
++
+.Example rebalancing template configuration 
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaRebalanceApiVersion}
+kind: KafkaRebalance
+metadata:
+  name: my-add-remove-brokers-rebalancing-template
+  annotations:
+    strimzi.io/rebalance-template: true # <1>
```

Comment:
This has to be `"true"` I think ...

```suggestion
    strimzi.io/rebalance-template: "true" # <1>
```

Reply:
This should be `strimzi.io/rebalance : template` as I pointed out in the previous comment